api/test/tests/controllers/meeting.js

```diff
@@ -204,6 +204,47 @@ describe( 'lib/controllers/meeting', () => {
       }
     });
 
+    it( 'should get a users owned and participating meetings', async() => {
```

Comment:
Last thing, this test looks good to me but can you put it in its own `describe( '#index' ` block?